src/styles/features/view/menuBar.css

```diff
@@ -33,6 +36,7 @@
 .title-container {
   padding-left: calc(1em - var(--icon-size) / 3);
   padding-right: 1em;
+
   color: var(--midnight-blue);
```

Comment:
Var names should be semantic, not the name of the colour.  What if in future you want to change the variable value and it's not blue?

Reply:
Names updated. They no longer use the name of the colour, instead using a descriptor of where the colour should be used.